app/scripts/modules/core/loadBalancer/filter/loadBalancer.filter.component.html

```diff
@@ -0,0 +1,92 @@
+<insight-filter>
```

Comment:
Example of a concrete filter (`load-balancer-filter`) using the `insight-filter` to render its chrome.